RadarSDK/RadarLogger.m

```diff
@@ -49,7 +49,12 @@ - (void)logWithLevel:(RadarLogLevel)level type:(RadarLogType)type message:(NSStr
 }
 
 - (void)logWithLevel:(RadarLogLevel)level type:(RadarLogType)type message:(NSString *)message includeDate:(BOOL)includeDate includeBattery:(BOOL)includeBattery append:(BOOL)append{
-     NSString *dateString = [self.dateFormatter stringFromDate:[NSDate date]];
+    RadarLogLevel logLevel = [RadarSettings logLevel];
+    if (level > logLevel) {
+        return;
+    }
```

Comment:
Hmmm, do we need to think through other effects of this?

Reply:
this check used to be performed inside dispatch_async after Radar sendLog, which means the logs are still sent to server. moving it up here prevents logs with higher value (less important) than the set level from being sent. This matches android logging behaviour.